src/CarbonAware.WebApi/test/integrationTests/CarbonAwareControllerTests.cs

```diff
@@ -248,65 +243,61 @@ public async Task EmissionsActual_ReturnsOk(DateTimeOffset start, DateTimeOffset
                 Assert.That(data, Is.Not.Null);
                 var value = await JsonSerializer.DeserializeAsync<CarbonIntensityDTO>(data);
                 Assert.That(value, Is.Not.Null);
-                Assert.That(value!.CarbonIntensity, Is.Not.EqualTo(0));
-                Assert.That(value!.StartTime, Is.EqualTo(start));
-                Assert.That(value!.EndTime, Is.EqualTo(end));
+                Assert.That(value!.CarbonIntensity, Is.Not.Null);
+                Assert.That(value!.StartTime, Is.EqualTo(startDate));
+                Assert.That(value!.EndTime, Is.EqualTo(endDate));
             }
         }
     }
 
-    [TestCase("location", "", TestName = "EmissionsActual empty location query string expects BadRequest")]
-    [TestCase("non-location-param", "", TestName = "EmissionsActual location param not present expects BadRequest")]
-    public async Task EmissionsActual_EmptyLocationQueryString_ReturnsBadRequest(string queryString, string value)
```

Comment:
nit: TestName can be made consistent with the other tests " 'methodName' expects BadRequest for missing parameters: "